README.md

```diff
@@ -1,2 +1,13 @@
 # speakbox-fall-2018
+
 Wiki: [Simplified GIT Workflow](https://github.com/redacademy/speakbox-fall-2018/wiki/Simplified-GIT-Workflow)
+
+## Technology Used
+
+Front End:
+
+- React Native
+
+Back End:
+
+- Realm
```

Comment:
The realm is not a backend. It is a database that you use with your app. Please make the changes.